src/components/display/Dropdown.tsx

```diff
@@ -421,12 +421,14 @@ const Dropdown = React.forwardRef<HTMLDivElement, DropdownProps>(function Dropdo
 
 	const handleClick = useCallback<(e: React.SyntheticEvent | KeyboardEvent) => void>(
 		(e) => {
-			if (!disabled && !openRef.current) {
-				e.preventDefault();
+			e.preventDefault();
```

Comment:
I'm afraid this change can break something since now preventDefault is always called even when the handler does nothing